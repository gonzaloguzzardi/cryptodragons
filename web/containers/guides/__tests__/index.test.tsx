import renderer from 'react-test-renderer'
import IndexPage from '..'

describe('Index page', () => {
  it('should match the snapshot', () => {
    const tree = renderer.create(<IndexPage deviceType="mobile" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
