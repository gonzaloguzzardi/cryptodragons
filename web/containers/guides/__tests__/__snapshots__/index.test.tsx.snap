// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page should match the snapshot 1`] = `
<div
  className="container"
>
  <main>
    <section
      className="jsx-683114997"
    >
      <div
        className="jsx-683114997 container"
      >
        <div
          className="jsx-683114997 header"
        >
          <h2
            className="jsx-683114997"
          >
            Welcome to CryptoDragons
          </h2>
        </div>
        <a
          aria-disabled={false}
          className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
          href="demo"
          onBlur={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
        >
          <span
            className="MuiButton-label"
          >
            Demo
          </span>
        </a>
      </div>
    </section>
  </main>
</div>
`;
